packages/maskbook/src/plugins/NFT/UI/NFTInPost.tsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from 'react'
+import MaskbookPluginWrapper from '../../MaskbookPluginWrapper'
+import type { NFTInPostProps, TokenDetails } from '../types'
+import { parseNftUrl } from '../utils'
+import NFTCardUI from './NFTCardUI'
+
+export default function NFTInPost(props: NFTInPostProps) {
+    const [tokenDetails, setTokenDetails] = useState<TokenDetails>({})
+
+    useEffect(function () {
+        const [address, tokenId] = parseNftUrl(props.nftUrl)
+
+        if (address && tokenId) {
+            const link = new window.URL('https://api.opensea.io/api/v1/assets')
```

Comment:
```suggestion
            const link = new URL('https://api.opensea.io/api/v1/assets')
```

Reply:
Oh sorry, missed that out. Will update in the next commit.